README.md

```diff
@@ -65,7 +65,7 @@ This goal downloads an API definition from SwaggerHub to a local file as part of
     <plugin>
         <groupId>io.swagger</groupId>
         <artifactId>swaggerhub-maven-plugin</artifactId>
-        <version>1.0.3-SNAPSHOT</version>
+        <version>1.0.3</version>
```

Comment:
[Stable Version](https://github.com/swagger-api/swaggerhub-maven-plugin/blob/master/README.md#stable-version) section needs to be updated too.